libs/entry-components/styles/modules/_default-theme.scss

```diff
@@ -50,6 +50,10 @@ $theme: (
 		)
 	),
 	tables: (
+		header: (
+			font-size: null,
+			font-weight: null
```

Comment:
Do we really need to support weight in the end?

Reply:
Perhaps it would be best to delay font-weight property for the time being, and revisit the idea if there is a need for it in future projects. Implementing it now would mean adding a property for the sorted state too, so it might be set aside for now if  @Cvetanovic  agrees?